### ISSUE TITLE
Unable to replicate results. 

### ISSUE THREAD

**Issue by nadarasarbahavan** [saurabhsharma1993/prototype-classifier-ltr#5]
I downloaded your code and also the code from LTR-weight-balancing github repo.
From the LTR-weight-balancing I used trained weight decay encoder as a fixed encoder in your code and yet i get results like this for the Balance factor=100 for CIFAR100 for the softmax classifier.

When comparing Table 2 in your paper:
My RUN: Many:        72.11 | Medium       53.60 | Few:        28.97 | All:        52.69
Claimed in the paper: Many:         76.12 | Medium     46.13 | Few:      11.34 | All:       46.12

Why is it there is a dramatic difference in the softmax results? I used the repo exactly as it is.

**Comment by saurabhsharma1993** [comment]
Not sure what you're doing without more context. 

Compare Table 2, softmax in the paper to Table 1, WD tuned (w/CE) in LTR-weight-balancing paper. 
There is no difference in our results from theirs.
